common/culture/cultures/wc_yaungolic.txt

```diff
@@ -6,6 +6,16 @@
 	language = language_taurahe
 	martial_custom = martial_custom_male_only
 	traditions = {
+		tradition_only_the_strong
+		tradition_warrior_culture
+		tradition_quarrelsome
+		tradition_horse_lords
```

Comment:
Better to relocalize it then since we have centaurs and not all cavalry are horses.

Reply:
How does this sound?

>  tradition_horse_lords_name:0 "Masters of the Plains"
 tradition_horse_lords_desc:0 "Whether upon the saddle or by ones own hooves, warriors of this culture specialise in striking foes with lighting speed and exploiting vulnerabilities in enemy formations to devastating effect."

Reply:
> > tradition_horse_lords_name:0 "Masters of the Plains"
> > tradition_horse_lords_desc:0 "Whether upon the saddle or by one's own hooves, warriors of this culture specialise in striking foes with lightning speed and exploiting vulnerabilities in enemy formations to devastating effect."

Sounds good.